problems/removeNumbersAtOddIndices.js

```diff
@@ -2,7 +2,13 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
+function removeNumbersAtOddIndices(arr) {
+    let output = arr.toString("");
+    for (i = 0; i < arr.length; i += 2)
+        output += arr[i];
+
+
```

Comment:
We need to return an array. This will return a string. You're also adding every other element onto a string that already has all the elements. 
Instead try using .filter for cleaner code. Remember the callback on array methods can also take the index as a second argument.